File: BooomJin/행렬의덧셈.py
def solution(arr1, arr2):
    answer = []
    result = []
    for i in range(len(arr1)):
        for j in range(len(arr1[0])):
            result.append(arr1[i][j]+arr2[i][j])
        answer.append(result)
        result = []
    return answer

arr1 = [[1,2],[2,3]]
arr2 = [[3,4],[5,6]]

print(solution(arr1,arr2))

## 함수를 활용하자..
def sumMatrix(A,B):
    return [list(map(sum, zip(*x))) for x in zip(A, B)]

File: jiHyeong/행렬의 덧셈.cpp
/*
포인트
1. 2차원 배열
2. 배열의 자료형이 배열(vector)이다. vector<vector<int>>
*/

#include <string>
#include <vector>

using namespace std;

vector<vector<int>> solution(vector<vector<int>> arr1, vector<vector<int>> arr2) {
    vector<vector<int>> answer;
    int i = 0;
    int j = 0;

    for (i = 0; i < arr1.size(); i++) {
        vector<int> input;
        for (j = 0; j < arr1[i].size(); j++) {
            input.push_back(arr1[i][j] + arr2[i][j]);
        }
        answer.push_back(input);
    }
    return answer;
}